controllers/admin/config.go

```diff
@@ -75,6 +76,7 @@ func SetStreamTitle(w http.ResponseWriter, r *http.Request) {
 	}
 	if value != "" {
 		sendSystemChatAction(fmt.Sprintf("Stream title changed to **%s**", value), true)
+		go webhooks.SendStreamStatusEvent(models.StreamTitleUpdated)
```

Comment:
Moved this here to fix race condtions where the webhook data had the older stream title.